hack/clear-route-status.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+clear_status ()
+{
+	MY_JSON_BLOB=$(curl -s -X GET http://localhost:8001/oapi/v1/namespaces/$1/routes/$2/)
```

Comment:
Why aren't we using an actual client for this? If we NEED raw, at the least `oc get --raw`